compiler/rustc_span/src/lib.rs

```diff
@@ -68,9 +68,8 @@ use std::ops::{Add, Range, Sub};
 use std::path::{Path, PathBuf};
 use std::str::FromStr;
 
-use md5::Digest as Md5Digest;
+use md5::Digest;
```

Comment:
is this used?

Reply:
`x.py check` fails if this `use` is removed:

```
❯  x.py check
Updating only changed submodules
  Submodules updated in 0.10 seconds
Building rustbuild
    Finished dev [unoptimized] target(s) in 0.42s
Checking stage0 std artifacts (aarch64-apple-darwin -> aarch64-apple-darwin)
    Finished release [optimized] target(s) in 0.25s
Checking stage0 std test/bench/example targets (aarch64-apple-darwin -> aarch64-apple-darwin)
    Finished release [optimized] target(s) in 0.25s
Checking stage0 compiler artifacts (aarch64-apple-darwin -> aarch64-apple-darwin)
    Checking rustc_span v0.0.0 (/Users/repos/rust/compiler/rustc_span)
error[E0599]: no function or associated item named `digest` found for struct `CoreWrapper` in the current scope
    --> compiler/rustc_span/src/lib.rs:1301:45
     |
1301 |                 value.copy_from_slice(&Md5::digest(data));
     |                                             ^^^^^^ function or associated item not found in `CoreWrapper<Md5Core>`
     |
     = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
     |
32   | use md5::Digest;
     |

error[E0599]: no function or associated item named `digest` found for struct `CoreWrapper` in the current scope
    --> compiler/rustc_span/src/lib.rs:1304:46
     |
1304 |                 value.copy_from_slice(&Sha1::digest(data));
     |                                              ^^^^^^ function or associated item not found in `CoreWrapper<Sha1Core>`
     |
     = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
     |
32   | use md5::Digest;
     |

error[E0599]: no function or associated item named `digest` found for struct `CoreWrapper` in the current scope
    --> compiler/rustc_span/src/lib.rs:1307:48
     |
1307 |                 value.copy_from_slice(&Sha256::digest(data));
     |                                                ^^^^^^ function or associated item not found in `CoreWrapper<CtVariableCoreWrapper<Sha256VarCore, UInt<UInt<UInt<UInt<UInt<UInt<UTerm, B1>, B0>, B0>, B0>, B0>, B0>>>`
     |
     = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
     |
32   | use md5::Digest;
     |

For more information about this error, try `rustc --explain E0599`.
error: could not compile `rustc_span` due to 3 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
Build completed unsuccessfully in 0:00:07
```